run.sh

```diff
@@ -8,13 +8,13 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-if [ "$1" == "-prod" ]; then
+if [ "$1" = "-prod" ]; then
```

Comment:
Right. I assumed sh was a symlink to Bash on most systems, but == is not POSIX compatible.